README.md

```diff
@@ -1,2 +1,246 @@
-# budget-app
-This is a Ruby on Rails capstone project, a mobile web application for managing your budget. It lists categories of transactions and details of each transaction. to show how much money you spent and on what.  Users register and log in to make data private to them. 
+# Budget App: Ometman
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./ometman-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Budget App: by Ometman</b></h3>
+  <h5>Budget App is a beautiful Ruby on Rails mobile web application for managing your budget. Users who must login, can create expense categories and transactions specific to each.  The home shows categories specific to the logged in user and shows total amount from all categories. Clicking a category shows how much money you spent and on what, with specific total in that category.  Data is private to the logged in user. The power of rails is employed to build models, controllers and views that allows easy CRUD funtionality to be performed. Built on a simple Entity Relational Diagram it has a simple UI for a rich UX. 
+  What more? Enjoy it!
+</div>
+
+# 📗 Table of Contents
+
+- [Budget App: Ometman](#budget-app-ometman)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Ometman Budget App ](#-ometman-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install Packages and Run Tests ](#install-packages-and-run-tests-)
+    - [Usage ](#usage-)
+    - [Deployment](#deployment)
+    - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 Ometman Budget App <a name="about-project"></a>
+
+**A Demo Video: Introduction to this Project**
+--[Budget Demo](https://www.loom.com/share/7c5642f53ab6429cbbdd520937e72812?sid=3b5006c2-5d4e-4eb2-bcf0-ad366796f2f3)
+
+> The project is mobile-oriented designs for different screen sizes.
+
+- It is accessible on any screen.
+- Better view on Mobile Devices and Ipads.
```

Comment:
- [ ] Please make sure that the project is deployed and accessible online and add the live demo link in the README file as requested in the project requirements.